nbextensions/appCell2/widgets/tabs/resultsViewer.js

```diff
@@ -64,13 +64,6 @@ define([
                 ]);
                 container.innerHTML = layout;
 
-                ui.setContent('summary.body', p([
```

Comment:
this element doesn't exist, and the info is already available in the status line of the app cell ==> delete!